deploy/chart/ingressmonitorcontroller/values.yaml

```diff
@@ -0,0 +1,50 @@
+global:
+# labels added on all components of chart in addition to some default labels
+  labels: {}
+
+# if empty, IMC will look in all namespaces, 
+# you can restrict to single namespace
```

Comment:
We can specify multiple namespaces as well by adding comma separated list.

Reply:
ok, will try to fix this

Reply:
You just need to fix the comment, nothing else.

Reply:
wouldn't it be better to create role & rolebinding for all the namespaces mentioned?

Reply:
Yes, if you can that would be awesome :) In case if a person specifies namespace and the flag is not empty then we need to create `role, rb` instead of `clusterRole, CRB`

Reply:
yeah, thinking of doing that

Reply:
done, now if a user adds multiple namespaces comma separated, chart will create Role, RB in all those namespaces